src/posix/client.cpp

```diff
@@ -59,6 +59,12 @@
 
 #include "platform-posix.h"
 
+const size_t LINE_BUFFER_SIZE = 256;
```

Comment:
```suggestion
static const size_t kLineBufferSize = 256;
```

Reply:
any difference when using `static` ?

Reply:
`static` limits the visibility of the name inside current `.cpp` file.

Reply:
Make sense. Thanks.

Reply:
revised to use enum